packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -504,71 +311,6 @@ describe('TransactionController', () => {
       typeof PendingTransactionTracker
     >;
 
-  /**
-   * Create a new instance of the TransactionController.
-   *
-   * @param opts - Options to use when creating the controller.
-   * @param opts.options - Any controller options to override the test defaults.
-   * @param opts.config - Any configuration to override the test defaults.
-   * @param opts.network - The mock network to use with the controller.
-   * @param opts.approve - Whether transactions should be immediately approved.
-   * @param opts.reject - Whether transactions should be immediately rejected.
-   * @param opts.state - The initial state to use for the controller.
-   * @returns The new TransactionController instance.
-   */
-  function newController({
```

Comment:
Minor question, but I've always done helper functions under constants at the top of test files based on pre-existing unit tests I first looked at.

Is the "official" standard to put helpers at the bottom of the file now?

Reply:
It's a practice I've been following in new files because I find it annoying to have to scroll down to find the opening `describe`. But it's definitely an unofficial standard.

I've moved these functions back up to make it easier to tell that `setupController` replaces `newController`.